doc/governance/open_network.rst

```diff
@@ -38,9 +38,9 @@ Other members are then allowed to vote for the proposal, using the proposal id r
         }
     }
 
-    $ scurl.sh https://<ccf-node-address>/gov/proposals/5/votes --cacert network_cert --key member1_privk --cert member1_cert --data-binary @vote_accept.json -H "content-type: application/json"
+    $ scurl.sh https://<ccf-node-address>/gov/proposals/f665047e3d1eb184a7b7921944a8ab543cfff117aab5b6358dc87f9e70278253/votes --cacert network_cert --key member1_privk --cert member1_cert --data-binary @vote_accept.json -H "content-type: application/json"
```

Comment:
How many bytes of this digest do we need to keep? Could we truncate these IDs to ~12 chars so they're more readable in these URLs?
(If we wanted to get fancy I think we could actually do a git-style thing where the public ID is "as few chars as possible while still unique", down to a minimum of N chars, by checking against each existing proposal ID when we generate a new one...)

Reply:
These IDs need to be properly unique, collisions are a security issue, so it's not obvious to me that there is any scope to truncate them. Unlike git, we very much want not to keep (or access) the whole history of proposals ids and have to scan it to find collisions.

Reply:
Git seems to scan for collisions when picking the short sha: https://lkml.org/lkml/2013/9/30/365, which isn't something we want to have to do. If we don't do that, we increase the chance of collision.